src/test/run-pass/mir_trans_calls.rs

```diff
@@ -88,6 +88,19 @@ fn test8() -> isize {
     Two::two()
 }
 
+#[link(name = "c")]
+extern {
+    fn printf(format: *const u8, ...) -> i32;
```

Comment:
hmm, is there something other than `printf` that we could test here?

I mean, I guess its fine (if a little opaque) to rely on the return value of `printf` as the basis for the test; but then again, that's relying on them never returning an error code ... seems like there are more direct options here, like `strlen`, or `qsort`,  (well `qsort` might be a bit of a stretch).

If you are deliberately trying to test the handling of varargs functions, then maybe go with `sscanf`?  Basically I'm trying to avoid doing IO where the output isn't actually checked for correctness, since that might mask bugs...